docs/source/posts/2021/2021-06-13-week-2-antriksh.rst

```diff
@@ -0,0 +1,26 @@
+Week #2: Feature additions in UI and IO modules
+===============================================
+
+.. post:: June 13 2021
+   :author: Antriksh Misri
+   :tags: google
+   :category: gsoc
+
+What did I do this week?
+------------------------
+This week I had to work on 3 PR's as well as some documentation. I really enjoyed this week's work as the tasks were really interesting. The aim for these PR's were to actually add a couple of features in the UI as well as the IO module, which includes, adding support for border in Panel2D, adding support for network/URL images in load_image method in IO module, adding resizing Panel2D from bottom right corner, completing the document with layout solutions provided by Unity/Unreal engine. Below are the PR's that I worked on:-
```

Comment:
wrong grammar - PR's, make it PRs

Reply:
Also, can you remove the `-` at the end of the paragraph

Reply:
Done.